docs/lock_status.md

```diff
@@ -16,8 +16,38 @@ when the lock is enabled and `False` otherwise.
 
 |Method                    |Description |
 |--------------------------|------------|
-|`locks.get_caps_lock() `  |Num Lock    |
```

Comment:
Is there a reason that caps_lock appears to have been replaced?

Reply:
There was a typo in the documentation. The method get_caps_lock was listed twice, once for caps lock and once for num lock. If you look at the side by side diff, you will see the change more clearly.